src/client/js/components/More.jsx

```diff
@@ -1,28 +1,84 @@
-import { NavLink } from 'react-router-dom'
-import { style } from 'typestyle'
-import { debugClassName } from '../styles/common'
-
-const buildLink = pathname => {
-  let pathParts = pathname.split('/')
-  let currentPage = parseInt(pathname.split('page/')[1])
-  let nextPage = Number.isInteger(currentPage) ? currentPage + 1 : 2
-  if (!pathParts[1] || pathParts[1] === 'page') return `/page/${nextPage}`
-  return `/${pathParts[1]}/page/${nextPage}`
+import { Link } from 'react-router-dom'
+import { stylesheet, classes } from 'typestyle'
+import { colors } from '../styles/common'
+
+const buildLinks = pathname => {
+  const pathParts = pathname.split('/')
+  const currentPage = parseInt(pathname.split('page/')[1])
+  const nextPage = Number.isInteger(currentPage) ? currentPage + 1 : 2
+  const prevPage = Number.isInteger(currentPage) && currentPage > 2 ? currentPage - 1 : null
+
+  if (!pathParts[1] || pathParts[1] === 'page') {
+    return {
+      currentPage,
+      prevLinkEnabled: currentPage >= 2,
+      prevLink: Number.isInteger(currentPage) && currentPage > 2 ? `/page/${prevPage}` : '/',
+      nextLink: `/page/${nextPage}`
+    }
+  }
+
+  return {
+    currentPage,
+    prevLinkEnabled: currentPage >= 2,
+    prevLink: Number.isInteger(currentPage) && currentPage > 2 ? `/${pathParts[1]}/page/${prevPage}` : `/${pathParts[1]}`,
+    nextLink: `/${pathParts[1]}/page/${nextPage}`
+  }
 }
 
-export const moreLinkClassName = style(debugClassName('more-link'), {
-  marginBottom: rem(1),
-  padding: `0 0 0 ${rem(4.5)}`
+const styles = stylesheet({
+  more: {
+    padding: `1em 0 1em 0`,
+    background: colors.LIGHTEST_GRAY,
+    textAlign: 'right',
+    gridColumnStart: '1',
+    gridColumnEnd: '4',
+    gridRowStart: '2',
+    marginRight: '6px'
+  },
+  moreItem: {
+    background: colors.NEARFORM_BRAND_ACCENT_2,
+    marginLeft: '1em',
+    color: 'white',
+    cursor: 'pointer',
+    fontWeight: '800',
+    padding: '10px 12px',
+    borderRadius: '10px',
+    boxShadow: '2px 2px 8px 0px rgba( 0, 0, 0, 0.2 )'
+  },
+  moreDisabled: {
+    background: colors.GRAY,
+    cursor: 'not-allowed',
+    boxShadow: 'none'
+  }
 })
 
 export function More({ location }) {
-  let link = buildLink(location.pathname)
+  const {nextLink, prevLink, prevLinkEnabled, currentPage} = buildLinks(location.pathname)
+
+  const rangeStart = currentPage ? 30 * (currentPage - 1) + 1 : 1
+  const rangeEnd = rangeStart + 29
 
   return (
-    <p className={moreLinkClassName}>
-      <NavLink to={link}>Load more</NavLink>
-    </p>
+    <div className={styles.more}>
+      <span>{`${rangeStart} - ${rangeEnd}`}</span>
+
+      {prevLinkEnabled &&
+        <Link className={styles.moreItem} to={prevLink}>
+          {'<'}
+        </Link>
+      }
+
+      {!prevLinkEnabled &&
+        <span className={classes(styles.moreItem, styles.moreDisabled)}>
+          {'<'}
+        </span>
+
+      }
+
+      <Link className={styles.moreItem} to={nextLink}>
```

Comment:
should prevent next when no further items to display

Reply:
This would be quite a bit of work as we'll need to share state across components, perhaps one to use the Context API with, I'd suggest raising this as a separate issue though, what do you think?

Reply:
it makes me think the previous next components are not in the right place.
they should probably be either part of the Comments/Stories or maybe preferably part of a common container which takes either Comments or Stories.
without a global state approach (redux/raw context) these look to be the appropriate approaches to  have access and pass down props about the collection